public/components/custom_panels/redux/panel_slice.ts

```diff
@@ -21,11 +21,12 @@ import {
 } from '../../../../common/types/custom_panels';
 import { coreRefs } from '../../../framework/core_refs';
 import { SavedObject, SimpleSavedObject } from '../../../../../../src/core/public';
-import { isNameValid } from '../helpers/utils';
+import { isNameValid, samplePanelName } from '../helpers/utils';
 import {
   addMultipleVisualizations,
   addVisualizationPanel,
 } from '../helpers/add_visualization_helper';
+import { createDemoPanel } from '../../../../server/common/helpers/custom_panels/sample_panels';
```

Comment:
Should we move this to `/common/constants/utils`? Cause, the `createDemoPanel` function is defined in the `server` whereas it is being used in `public` component.

Reply:
Yes, +1, please relocate `createDemoPanel` fn

Reply:
It's also being used in server here: https://github.com/opensearch-project/dashboards-observability/blob/72def7dde74c2611b5144fc65603e65ec814acd5/server/adaptors/custom_panels/custom_panel_adaptor.ts#L9